wordpress-mobile/AztecEditor-iOS
# FormatBar: Update icons

This PR adds alternative icons to the format bar for headings and list styles. To do this, `FormatBarItems` can now take a list of alternative icons to display for different formatting identifiers.

To test:

* Build and run, and go to the editor with sample content.
* Scroll to the **Lists** section. 
* Tap in the unordered list text, and check the lists icon in the format bar shows an unordered list.
* Tap in the ordered list text, and check the lists icon in the format bar shows an ordered list.
* Scroll to the **Headings** section.
* Tap on each of the heading sizes in turn, and check that the headings icon updates accordingly.

Needs review: @jleandroperez
